src/doc/telio_events.md

```diff
@@ -1,37 +1,350 @@
 # Documentation For Describing Telio Events
 
-This module documents the events reported by the Telio library.
+This module documents the events reported by the Telio library. The events are reported strongly-typed in the supported languages. The body of the event variants are described in later sections with the top level event being defined as:
+
+<multi-code-select></multi-code-select>
```

Comment:
I thinkg this was an overkill, it should be enough to point to Node in rust code, and people can look everything up in bindings, keeping structs here will just lead for this are to be constatntly out of sync with actual code.

So just leave textual info :sweat_smile: